topics/galaxy-interface/tutorials/workflow-posters/tutorial.md

```diff
@@ -0,0 +1,271 @@
+
+Now that we have a view of our Workflow without the Galaxy UI, let's make a high-res screenshot using Firefox.
+
+<div class="Print" markdown="1">
+Firstly position your Workflow, so all of it is visible in the way you would like it to be on the poster.
+</div>
+
+<div class="Web" markdown="1">
+Firstly position your Workflow, so all of it is visible in the way you would like it to be on the screenshot.
+</div>
+
+> <tip-title>Can't fit your entire Workflow at once?</tip-title>
+> 
+> If you zoomed out as far as possible, but can't view all of your Workflow, try the following:
+> * Navigate back to the full Workflow preview page (without the UI hidden)
+> * Click anywhere outside of the Workflow preview (e.g. on the about section)
+> * Zoom out the entire browser window a bit, by using <kbd>Ctrl</kbd> + <kbd>Mouse Wheel</kbd>, or <kbd>Ctrl</kbd> + <kbd>-</kbd>
+> * Re-add the query parameters we added in the previous section
+> * If you still can't fit all of it, repeat the above steps
+{: .tip}
+
+> <hands-on-title>Creating a high-resolution screenshot</hands-on-title>
+> 
+> 1. **Open the Firefox Javascript console.** To do this, either:
+>    - Use the shortcut <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>K</kbd> (On OS X <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>K</kbd>)
+>    - Or navigate to the console using your mouse:
+>       * Menu
+>       * More Tools
+>       * Web Developer Tools
+>       * Console Tab
+>
+> 2. **Resize or move the console**
+>
+>    The console may be in the way of your Workflow.
+>
+>    You can make it take up less space by dragging down the bar at the top.
+>
+>    You can also reposition it to the side, by selecting the three dots in the upper right corner of the console,
+>    and choosing a different position from: "Dock to Bottom", "Dock to Right", "Dock to Left", "Separate Window".
+>
+> 3. **Make the screenshot**
+>
+>    In the Javascript console type:
+>    ```
+>    :screenshot --dpr 6
+>    ```
+>    After a short moment, a new screenshot should appear in your downloads folder.
+>
+>    The value "6" after the `--dpr` option is just an example. Keep reading to learn how to choose the right dpr value.
+>
+{: .hands_on}
+
+<div class="Print" markdown="1">
+> <tip-title>Choosing the right dpr value for print</tip-title>
+>
+> `dpr` stands for **D**evice **P**ixel **R**atio.
+>  
+> This is how much Firefox will scale up your screenshot. A dpr of 2, will make your screenshot twice as wide and high as you browser window.
+> 
+> Always go higher than you think you need, when choosing a dpr for print.
+> Here's a rule-of thumb on how to determine the correct dpr:
+>
+> * Think about how much bigger your poster is going to be than your browser window, in physical dimensions.
+>   If unsure, round up.
+> * Multiply that number by 2
+>
+> Example: If your poster will be roughly 4 times larger than your browser window, choose a dpr of 8.
+>
+{: .tip}
+</div>
+
+<div class="Web" markdown="1">
```

Comment:
you should be able to use these inside the tip box, if it helps reduce duplication (just ensure newlines before/after to trigger `<p>`s in the rendered html

Reply:
Ah, I did not add any newlines before I think. Thanks, I'll re-structure some boxes using this!